web-client/public/js/update-app.js

```diff
@@ -933,10 +924,13 @@ export const updateApp = grnState => {
                 setTimeout(() => updaters.renderNodeColoring(), 250);
 
             }
+            if (grnState.mode === NETWORK_PPI_MODE) {
+              displayPPINodeColorWarning();
+            }
         }
+        refreshApp();
```

Comment:
As you might have observed, “refresh” calls are not expressly needed in frameworks like React and SwiftUI—but they still do happen. It is, in fact, part of what these frameworks offer that certain operations are done automatically and at the right moments, rather than leaving it to the programmer to make it happen